src/utilities.h

```diff
@@ -430,4 +432,63 @@ bool isStandardPrefixName(const std::string &name);
  */
 size_t getVariableIndexInComponent(const ComponentPtr &component, const VariablePtr &variable);
 
+/**
+ * @brief Trim whitespace from the front of a string (in place).
+ *
+ * Remove whitespace from the front of a string, modifying the passed string.
+ *
+ * @param s The @c std::string to trim.
+ */
+static inline void ltrim(std::string &s)
```

Comment:
This probably ought to be renamed `lTrim()` or `leftTrim()`?

Reply:
Done